app/views/categories/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render "shared/nav_bar"%>
+<%= render "shared/messages"%>
+<% @categories.each do |item|%>
+  <%= link_to category_path(item), class: 'text-decoration-none' do%>
+    <div class="row align-items-center m-1 p-1 border border-secondary">
+      <%= image_tag(item.icon_url, class: 'img-fluid col-4')%>
+      <p class= "col-4 text-center text-dark fs-5"><%= item.name%></p>
+      <p class= "col-4 text-center text-dark fs-5"><%=number_to_currency(item.transacctions.sum(:amount))%></p>
+    </div>
+  <%end%>
+<%end%>
+
+<%= link_to 'Add category', new_category_path, class:'my-3 btn btn-primary w-100 rounded-0' %>
```

Comment:
- [ ] Kindly improve your styling here. Your variation of the blue color in the header is inconsistent with the design. Kindly use the correct color variation to match the design. 

- [ ] The "Add category" button should be fixed at the button based on the design. Kindly implement that as well.

- [ ] Let's also try to maintain the design colors as much as possible. The button should have the greenish color of the design with a rectangular inner white border. Kindly implement that as well.

- [ ] The button should also be somewhat larger than it currently is. Kindly fix that.

- [ ] The font styling should also be maintained. Kindly style category cards and improve their fonts to match the design.

- [ ] Kindly add the greyish background as we have in the design also.

  Your Implementation | Behance Design
  :---:|:---:
  ![urc](https://user-images.githubusercontent.com/36542986/220814443-319edf86-9349-4084-9064-60fdf4a752e7.jpg) | ![riew](https://user-images.githubusercontent.com/36542986/220813033-9bc981b6-c654-4408-be0c-06cf2ff9c763.jpg)